spring-cloud-task-samples/timestamp/README.adoc

```diff
@@ -0,0 +1,26 @@
+= Timestamp Job
+
+This is Spring Cloud Task boot application that logs a timestamp .
```

Comment:
"Spring Cloud Task boot" doesn't sound right.  I think the right way to put this is that it's a Spring Boot application that uses Spring Cloud Task.